### PR TITLE
Week 3 walkthrough part 5

### PR DESCRIPTION
**Pull request by charmychancey**
implement datastore
- change indenting style from +4 spaces to +2, google styles
- part 1 of Datastore: store entities in database
- part 2 of Datastore: load entries from database

### CODE REVIEW

File: portfolio/src/main/webapp/about-me.html [hunk 1]
```diff
@@ -1,95 +1,95 @@
...
+              </li>
+              <li>
+                Most Hmong-Americans live in either California or Minnesota.
+              </li>
+              <li>
+                I became interested in computer science because of tech-savvy
+                figures like Elon Musk, and his passion to accelerate mankind's journey
+                into the future!I became interested in computer science because of tech-savvy
+                figures like Elon Musk, and his passion to accelerate mankind's journey
+                into the future!
```
Comment (snazzorama) [inline portfolio/src/main/webapp/about-me.html:79]: Looks like some text was duplicated here?

Comment (charmychancey) [reply portfolio/src/main/webapp/about-me.html:77]: Oops, fixed!

Comment (charmychancey) [reply portfolio/src/main/webapp/about-me.html:77]: ummm, what was the acronym again? :D

Comment (charmychancey) [reply portfolio/src/main/webapp/about-me.html:77]: Ready for another look

**Comment by snazzorama** [comment]
The acronym is "PTAL". :-)

**Pull request merged**